pkg/addons/applier.go

```diff
@@ -157,6 +165,7 @@ func newAddonsApplier(s *state.State) (*applier, error) {
 		Credentials:                         creds,
 		CSIMigration:                        csiMigration,
 		CSIMigrationFeatureGates:            csiMigrationFeatureGates,
+		OpenStackCCMClusterName:             openstackClusterName,
```

Comment:
I really would like to see this function to stay provider-agnostic. Such functions probably should be part of LiveCluster, and we may even expose it to the templates and have some method like `LiveCluster.CCMStatus.ComputedClusterName()`